Gems/Atom/RHI/Vulkan/Code/Source/RHI/PhysicalDevice.cpp

```diff
@@ -261,7 +261,7 @@ namespace AZ
                 VK_KHR_SHADER_FLOAT_CONTROLS_EXTENSION_NAME
             } };
 
-            [[maybe_unused]] uint32_t optionalExtensionCount = sizeof(optionalExtensions) / sizeof(VK_EXT_SAMPLE_LOCATIONS_EXTENSION_NAME);
+            [[maybe_unused]] uint32_t optionalExtensionCount = static_cast<uint32_t>(optionalExtensions.size());
```

Comment:
I think this was fixed recently so you will have to do a merge.

Reply:
yep, resolved the conflict and merged